### PR TITLE
Geiger Counter

### PR DESCRIPTION
**Pull request by Trickx**
A Geiger-Counter-Sensor-Module based on Event-Sensor.
Documentation is found in the Wiki page:
https://github.com/Trickx/espurna/wiki/Geiger-Counter

### CODE REVIEW

File: code/espurna/sensor.ino [hunk 1]
```diff
@@ -59,6 +59,7 @@ unsigned char _magnitudeDecimals(unsigned char type) {
         type == MAGNITUDE_POWER_REACTIVE) {
         if (_sensor_power_units == POWER_KILOWATTS) return 3;
     }
+    if (type == MAGNITUDE_GEIGER_SIEVERT) return 4;        // TODO: Is this required? "magnitude_decimals" are defined in progmem.h
```
Comment (xoseperez) [inline code/espurna/sensor.ino]: No, it is not, this method is here to handle non-default units (like KWh).

**Comment by xoseperez** [comment]
Merged, may I also clone you wiki page here?

**Comment by Trickx** [comment]
Hi Xose,

yes, feel free to clone the wiki, please.
It make sense to have all information at one place.
It would be cool to be mentioned somewhere.

Thanks,
Sven (aka Trickx)

**Comment by xoseperez** [comment]
@Trickx Sure! I always give attribution to contributors. Thank you!

**Comment by Trickx** [comment]
You are welcome!

**Pull request merged**